### ISSUE TITLE
alt Attributes

### ISSUE THREAD

**Issue by jordanwillis** [andykolk/teacozy#3]
Another thing to remember is to always include the `alt` attribute on your `<img>` elements.  This is important because this is one of many ways to improve accessibility and is the mechanism by which screen readers use to represent the image to users who cannot see.

https://github.com/andykolk/teacozy/blob/master/index.html#L11
https://github.com/andykolk/teacozy/blob/master/index.html#L38
etc.
